lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -112,21 +112,16 @@ function Test-ProjectPackage {
   param ($Path, $Algorithm = 'SHA256', $Hash)
 
   if (-not (get-command get-filehash -ErrorAction 'SilentlyContinue')) {
-    function disposable($o){($o -is [IDisposable]) -and (($o | get-member | foreach-object {$_.name}) -contains 'Dispose')}
-    function use($obj, [scriptblock]$sb){try {& $sb} catch [exception]{throw $_} finally {if (disposable $obj) {$obj.Dispose()}} }
-    function Get-FileHash ($Path, $Algorithm) {
-      $Path = (resolve-path $path).providerpath
-      $hash = @{Algorithm = $Algorithm; Path = $Path}
-      use ($c = Get-SHA256Converter) {
-        use ($in = (gi $path).OpenRead()) {
-          $hash.Hash = ([BitConverter]::ToString($c.ComputeHash($in))).Replace("-", "").ToUpper()
-        }
-      }
-      new-object PSObject -Property $hash
-    }
+    $hash = Get-FileHash $Path $Algorithm
+    new-object PSObject -Property $hash
```

Comment:
renaming `$hash` to `$DesiredHash` would make this a little clearer. this felt like cruft until I realized that.